ng-dev/pr/rebase/cli.ts

```diff
@@ -15,16 +15,23 @@ import {rebasePr} from './index.js';
 /** The options available to the rebase command via CLI. */
 export interface RebaseOptions {
   pr: number;
+  i?: boolean;
 }
 
 /** Builds the rebase pull request command. */
 function builder(argv: Argv): Argv<RebaseOptions> {
-  return addGithubTokenOption(argv).positional('pr', {type: 'number', demandOption: true});
+  return addGithubTokenOption(argv)
+    .positional('pr', {type: 'number', demandOption: true})
+    .option('i', {
+      type: 'boolean',
+      demandOption: false,
+      describe: 'Do the rebase interactively so that things can be squashed and amended',
+    });
```

Comment:
Can we have the option be the full word `interactive`, we can keep the `-i` as an alias flag.

```suggestion
    .option('interactive', {
      type: 'boolean',
      alias: ['i'],
      demandOption: false,
      describe: 'Do the rebase interactively so that things can be squashed and amended',
    });
```

Reply:
Ooo yes, definitely. I didn't know you could do this in the tooling. :)

Reply:
Done!